### ISSUE TITLE
Using native OCaml C stubs with compiled languages?

### ISSUE THREAD

**Issue by Lupus** [jordwalke/rehp#32]
I'm still dreaming about Go backend, and got an idea - since Golang interfaces nicely with C, may be there is little benefit in reimplementing C stubs in Go, but instead bind them and call directly? Two garbage-collected runtimes in one process, what could go wrong? :)

**Comment by jordwalke** [comment]
Yeah, two garbage collectors tend to be very difficult to integrate with each other. However garbage collectors and C also do not interop well together. Nothing interops well together :D
